src/flows_src.rs

```diff
@@ -56,7 +56,10 @@ impl<'a> TopLevelSrc<'a> {
     }
 
     fn check_get_node_to_relocate(&mut self) {
-        if let (Some(candidate), false) = (self.0.action.get_node_to_relocate(), false) {
+        if self.0.action.has_relocating_node() {
```

Comment:
I would probably have kept the tuple check it seemed closer to the flow.
Can you add a test like other existing one for this.

Reply:
I added the test.

I think the order of the parts of the `if` statement is an implementation detail, and this code is more readable to me.  I can change the order in the html doc if you prefer?

Reply:
fine either way.